AnkiDroid/src/main/java/com/ichi2/anki/CrashReportService.kt

```diff
@@ -281,12 +288,61 @@ object CrashReportService {
             }
         }
         if (FEEDBACK_REPORT_NEVER != reportMode) {
+            if (processThrowable(e) == null) {
+                return
+            }
             ACRA.errorReporter.putCustomData("origin", origin ?: "")
             ACRA.errorReporter.putCustomData("additionalInfo", additionalInfo ?: "")
             ACRA.errorReporter.handleException(e)
         }
     }
 
+    fun Throwable.replaceMessage(newMessage: String?): Throwable =
```

Comment:
This obscures the actual type of exception in logcat as it's converted to a Throwable. I believe this also loses the original stack trace, but I haven't confirmed.


```suggestion
    private fun Throwable.replaceMessage(newMessage: String?): Throwable =
```